app/services/subcase-is-approved.js

```diff
@@ -0,0 +1,30 @@
+import Service, { inject as service } from '@ember/service';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class SubcaseIsApprovedService extends Service {
+  @service store;
+
+  async approved(subcase) {
+    const meeting = await subcase.requestedForMeeting;
+
+    if (meeting?.isFinal) {
+      const approvedDecisionResultCode = await this.store.findRecordByUri(
+        'decision-result-code',
+        CONSTANTS.DECISION_RESULT_CODE_URIS.GOEDGEKEURD
+      );
+      const acknowledgedDecisionResultCode = await this.store.findRecordByUri(
+        'decision-result-code',
+        CONSTANTS.DECISION_RESULT_CODE_URIS.KENNISNAME
+      );
+
+      return !!(await this.store.queryOne('agenda-item-treatment', {
+        'filter[subcase][id]': subcase.id,
```

Comment:
Better to use `:id:` (since `id` is not officially supported by mu-cl-resources)